authorization/src/main/resources/config/liquibase/changelog/20230424-create-authorization-table.xml

```diff
@@ -37,25 +37,28 @@
 
     <changeSet author="Ritense" id="2">
         <addColumn tableName="permission">
-            <column name="role_key" type="VARCHAR(256)">
+            <column name="role_id" type="${uuidType}">
                 <constraints nullable="false"/>
             </column>
         </addColumn>
     </changeSet>
 
     <changeSet author="Ritense" id="3">
         <createTable tableName="role">
-            <column name="key" type="VARCHAR(256)">
+            <column name="id" type="${uuidType}">
                 <constraints nullable="false" primaryKey="true"/>
             </column>
+            <column name="key" type="VARCHAR(256)">
+                <constraints nullable="false" unique="true"/>
+            </column>
         </createTable>
     </changeSet>
 
     <changeSet author="Ritense" id="4">
-        <addForeignKeyConstraint baseTableName="permission" baseColumnNames="role_key"
-                                 constraintName="fk_permission_role_key"
+        <addForeignKeyConstraint baseTableName="permission" baseColumnNames="role_id"
```

Comment:
This is going to break our test environment, as the hash is changed. So either create a new changeset, add the hash to the changeset, or be prepared to do some manual db fixing.

Reply:
I don't think we even have anything deployed to the test environment, so that's not a concern right now (?)

Reply:
You're right.